readthedocs/audit/models.py

```diff
@@ -37,6 +38,12 @@ def new(self, action, user=None, request=None, **kwargs):
             kwargs.setdefault('resource', request.path_info)
             kwargs.setdefault('auth_backend', get_auth_backend(request))
 
+            # Fill the project from the request if available.
+            # This is frequently on actions generated from a subdomain.
+            project_slug = getattr(request, 'slug', None)
```

Comment:
I think we should use `request.host_project_slug` here instead. IIRC, `request.slug` is "kind of deprecated" and we replaced it by `.host_project_slug` when we built El Proxito.